site-content/source/modules/ROOT/pages/events/organizer-handbook.adoc

```diff
@@ -0,0 +1,106 @@
+* Speakers don’t have to be active in the local community, and they can be from out-of-town, but familiarity with existing community members generally means they will be more well-received than others.
+* Sessions can be presented in any format that is relevant or useful for the topic - we encourage you to be creative and mix up your event formats.
+* Slides must use a plain Cassandra deck that doesn’t include excessive vendor info and branding - if you aren’t sure whether slides are appropriate, please contact Planet Cassandra staff to ask about it.
+
+=== Finding Sponsors
+
+If you need a sponsor for your events, any company can come on board as a sponsor for your local Cassandra meetup; however, it is best to select sponsors that offer relevant products or services to Cassandra community members. Sponsors can come on board to cover any financial costs related to your events - venue hire, refreshments, etc. - or they can provide material sponsorship - venue, workshop materials, etc.
+
+In return for sponsorship, sponsors should receive some benefits. Some ideas for this are:
+
+* Promotion in event communications
+* Branding posted at the event
+* Discount codes for products/services provided to attendees
+* Distribution of company swag
+
+Organisers can be creative with the benefits they provide to sponsors, but please bear in mind the following non-negotiable restrictions:
+
+* Speaking spots are not provided to sponsors in return for their sponsorship under any circumstances.
+* A list of attendee names or contact details may not be given to sponsors (or anyone else) - aggregate demographic data is fine to share, but specific details about attendees are not.
+
+=== Finding a Venue
+
+Organisers can use any venue that is appropriate for the event and the people attending it. Here are some things to consider when selecting a venue:
+
+* Ensure that the venue has adequate accessibility features for all potential attendees.
+* The venue should be able to accommodate the number of expected attendees with seating for everyone, as well as include any features that are required for the format of the event (e.g. desks/tables for workshops that require attendees to use their laptops)
+* Avoid venues that could be potentially problematic for some attendees - this could include political or religious venues.
+
+=== Running Online Events
+
+Organisers may choose to run online events. The same guidelines apply to online events as to in-person ones. When selecting a platform to use for online events, ensure you select a platform that has adequate accessibility features for all attendees, as well as any features that the event format requires.
+
+=== Producing Swag
+
+You may produce an event or community-related swag for your community members at your discretion. While there are no restrictions on this, here are some guidelines to help organisers make good choices:
```

Comment:
this isn't correct.

swag does (should) be checked with the PMC and ASF Branding to ensure the ASF Trademarks are used correctly.

(the same also applies to all event materials, e.g. social graphics, flyers, banners, etc)